client/src/spotify/index.js

```diff
@@ -262,7 +262,14 @@ export const getTrackAudioFeatures = trackId =>
 
 export const getUserInfo = () =>
   axios
-    .all([getUser(), getFollowing(), getNowPlaying(), getPlaylists(), getTopArtistsLong(), getTopTracksLong()])
+    .all([
+      getUser(),
+      getFollowing(),
+      getNowPlaying(),
+      getPlaylists(),
+      getTopArtistsLong(),
+      getTopTracksLong(),
+    ])
```

Comment:
oops, this was vs code auto formatting >.< my bad. I tried to save without auto formatting when I remembered but I forgot here 😐